plugins/destination/snowflake/client/migrate.go

```diff
@@ -94,16 +93,16 @@ func (c *Client) autoMigrateTable(_ context.Context, table *arrow.Schema) error
 	return nil
 }
 
-func (c *Client) createTableIfNotExist(_ context.Context, table *arrow.Schema) error {
+func (c *Client) createTableIfNotExist(_ context.Context, table *schema.Table) error {
 	var sb strings.Builder
 	// TODO sanitize tablename
-	tableName := schema.TableName(table)
+	tableName := table.Name
```

Comment:
don't create extra var

Reply:
I didn't want to create more changes for now.